app-kotlin/build.gradle

```diff
@@ -1,47 +1,72 @@
 buildscript {
   repositories {
-    jcenter()
+    mavenCentral()
     google()
   }
 
   dependencies {
-    classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.0'
+    classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.10'
   }
 }
 
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-kapt'
+apply plugin: 'org.jetbrains.kotlin.android'
+
 
 android {
-  compileSdkVersion 31
+  compileSdkVersion 33
 
   defaultConfig {
     applicationId 'au.com.gridstone.trainingkotlin'
     minSdkVersion 21
-    targetSdkVersion 31
+    targetSdkVersion 33
     versionCode 1
     versionName '1.0'
   }
 
   buildTypes {
     release {
       minifyEnabled false
-      proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
     }
   }
 
-  compileOptions {
-    sourceCompatibility JavaVersion.VERSION_1_8
-    targetCompatibility JavaVersion.VERSION_1_8
-  }
-  kotlinOptions {
-    jvmTarget = '1.8'
+  sourceSets {
+    main.java.srcDirs += 'src/main/kotlin'
   }
 }
 
 dependencies {
-  implementation 'androidx.core:core-ktx:1.7.0'
   implementation 'androidx.appcompat:appcompat:1.4.2'
-  implementation 'com.google.android.material:material:1.6.1'
-  implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
+
+//  Support libraries
+  implementation 'androidx.appcompat:appcompat:1.4.2'
+  implementation 'androidx.recyclerview:recyclerview:1.2.1'
+
+//  Conductor
+  implementation 'com.bluelinelabs:conductor:3.1.5'
+
+// Lifecycle-aware Controllers (architecture components):
+  implementation 'com.bluelinelabs:conductor-archlifecycle:3.1.5'
+
+//  For retrofit / networking
+  implementation 'com.squareup.retrofit2:retrofit:2.9.0'
+  implementation 'com.squareup.retrofit2:converter-moshi:2.9.0'
+  implementation 'com.squareup.okhttp3:okhttp:5.0.0-alpha.1'
+
+//  Moshi
+  implementation 'com.squareup.moshi:moshi:1.13.0'
+  kapt 'com.squareup.moshi:moshi-kotlin-codegen:1.13.0'
```

Comment:
no longer needed i think

Reply:
Thanks for mentioning yes we don't need this anymore